cmd/cli/commands/master.go

```diff
@@ -21,19 +24,27 @@ var masterRootCmd = &cobra.Command{
 }
 
 var masterListCmd = &cobra.Command{
-	Use:   "list",
-	Short: "Show known worker's addresses",
-	Run: func(cmd *cobra.Command, _ []string) {
+	Use:    "list [master_eth]",
+	Short:  "Show known worker's addresses",
+	PreRun: loadKeyStoreIfRequired,
+	Run: func(cmd *cobra.Command, args []string) {
 		ctx, cancel := newTimeoutContext()
 		defer cancel()
 
+		var master common.Address
+		if len(args) > 0 {
+			master = common.HexToAddress(args[0])
```

Comment:
we should validate that given value is valid address using `common.IsHexAddress(s)`